### DISCUSSION TITLE
c3105 script

### DISCUSSION THREAD

**Discussion by layssi** [JSBSim-Team/jsbsim#1156] [General]
The script --script scripts/c3105.xml is supposed to follow waypoints but it crashes

![Screenshot from 2024-09-09 09-25-22](https://github.com/user-attachments/assets/69bbafbc-1b92-410d-8893-4ddbff6da151)

**Comment by layssi** [comment]
Can not get a functional waypoint Navigator

**Comment by seanmcleod** [comment]
Have you taken an actual look at the `c3105.xml` script? It may well navigate between some waypoints, but it also looks like the script is written to cause and expect a crash!

https://github.com/JSBSim-Team/jsbsim/blob/e607c278cb8c472ac7bf417db9368acfc8c7c1f8/scripts/c3105.xml#L58-L96
